akka-docs/rst/scala/code/docs/actor/FSMDocSpec.scala

```diff
@@ -16,17 +16,17 @@ class FSMDocSpec extends MyFavoriteTestFrameWorkPlusAkkaTestKit {
 
   //#fsm-code-elided
   //#simple-imports
-  import akka.actor.{ Actor, ActorRef, FSM }
+  import akka.actor.{ ActorRef, FSM }
   import scala.concurrent.duration._
   //#simple-imports
   //#simple-events
   // received events
-  final case class SetTarget(ref: ActorRef)
-  final case class Queue(obj: Any)
+  case class SetTarget(ref: ActorRef)
```

Comment:
Please keep the final keywords on case classes. We use those because extending case classes is not safe generally speaking and by sealing the inheritance here we avoid "by accident extending it".